src/libs/actions/Report.js

```diff
@@ -648,7 +649,8 @@ function reconnect(reportID) {
                     key: `${ONYXKEYS.COLLECTION.REPORT}${reportID}`,
                     value: {
                         isLoadingReportActions: true,
```

Comment:
NABish but can we rename this to `isLoadingInitialReportActions`?

Reply:
yes, then it would make more sense, will do this